swift_scality_backend/diskfile.py

```diff
@@ -548,3 +553,11 @@ def delete(self, timestamp):
                           called externally only by the `ObjectController`
         """
         self._filesystem.del_object(self._name)
+
+    @property
+    def timestamp(self):
+        if self._metadata is None:
+            raise swift.common.exceptions.DiskFileNotOpen()
+        return swift.common.utils.Timestamp(self._metadata.get('X-Timestamp'))
```

Comment:
I'll go with your approach.

>So, if swift.common.utils.Timestamp doesn't exist this code would fail.
If it doesn"t exist, that code won't be called in the first place.